cloud/jenkins/psmdb_operator_gke_latest.groovy

```diff
@@ -401,7 +406,7 @@ pipeline {
             description: 'GKE kubernetes version',
             name: 'PLATFORM_VER')
         choice(
-            choices: 'rapid\nstable\nregular',
+            choices: 'stable\nregular\nrapid\nNone',
```

Comment:
This will make stable default one. We need rapid to stay default.

Reply:
done